js/background.js

```diff
@@ -325,4 +252,4 @@ function AppendThumbnail(tabId, cahngeInfo, tab)
 	);
 }
 
-chrome.tabs.onUpdated.addListener(AppendThumbnail);
+chrome.tabs.onUpdated.addListener(AppendThumbnail);
```

Comment:
There are multiple updates in a tab lifecycle. For its status, it go from unloaded to loading to complete, for example. I suppose we want to capture only the screen of a loaded page update, so this should be more efficient.

```suggestion
chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) =>  {
	if (changeInfo.status === "complete")
		AppendThumbnail(tabId, changeInfo, tab)
});
```

Reply:
Seems legit)